app/react/V2/Routes/Settings/IX/components/FiltersSidepanel.tsx

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { Translate, t } from 'app/I18N';
+import { Button, Card, Sidepanel } from 'V2/Components/UI';
+import { Checkbox } from 'app/V2/Components/Forms';
+import { useForm } from 'react-hook-form';
+import { useSearchParams } from 'react-router-dom';
+import { SuggestionCustomFilter } from 'shared/types/suggestionType';
+
+interface FiltersSidepanelProps {
+  showSidepanel: boolean;
+  setShowSidepanel: React.Dispatch<React.SetStateAction<boolean>>;
+  aggregation: any;
+}
+
+const header = (label: string, total: number) => {
+  return (
+    <div className="flex items-center space-x-2 text-indigo-700">
+      <div className="flex-none">{label}</div>
+      <div className="flex-1 border-t border-dashed border-t-gray-200" />
+      <div className="flex-none">{total}</div>
+    </div>
+  );
+};
+
+const FiltersSidepanel = ({
+  showSidepanel,
+  setShowSidepanel,
+  aggregation,
+}: FiltersSidepanelProps) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const defaultFilter: SuggestionCustomFilter = {
+    labeled: {
+      match: false,
+      mismatch: false,
+    },
+    nonLabeled: {
+      noContext: false,
+      noSuggestion: false,
+      obsolete: false,
+      others: false,
+    },
+  };
+
+  let initialFilters: SuggestionCustomFilter = defaultFilter;
+
+  try {
+    if (searchParams.has('filter')) {
+      initialFilters = JSON.parse(searchParams.get('filter')!);
+    }
+  } catch (e) {}
+
```

Comment:
I've seen a couple of places where we have this try catch statements where nothing happens on the catch. Could they be removed?

Reply:
I purposely put that there to allow JSON.parse to fail silently. For example, if a user manually edited filter query string to an invalid JSON string, uwazi should ignore the filters in the query string and use the default filters